src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+    int supplyCounter = 0;
+    int buyCounter = 0;
+        Path path = Path.of(fromFileName);
+        Path pathToSave = Path.of(toFileName);
+        try {
+            String readedFile = String.valueOf(Files.readAllLines(path)).
+                    replaceAll("[^sa-zA-Z0-9]", ",");
```

Comment:
The use of `replaceAll("[^sa-zA-Z0-9]", ",")` is incorrect for parsing the file content. This regex will replace any non-alphanumeric character with a comma, which might not correctly parse the file if it contains other delimiters or special characters. Consider using a more appropriate method to read and split the file content based on the actual delimiter used in the file.